fudy/CMakeLists.txt

```diff
@@ -8,10 +8,15 @@ set(CMAKE_AUTORCC ON)
 
 set(CMAKE_CXX_STANDARD 17)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
+set(CMAKE_TOOLCHAIN_FILE "${CMAKE_BINARY_DIR}/conan_toolchain.cmake")
+
+option(BUILD_SHARED_LIBS "Set the default type for add_library to STATIC" OFF)
```

Comment:
Change the description

Reply:
Could you please clarify for me what you mean?

Reply:
I updated this description.